resources/js/beatmap-discussions/new-discussion.tsx

```diff
@@ -455,6 +455,7 @@ export class NewDiscussion extends React.Component<Props> {
         props={{
           'data-type': type,
           onClick: this.post,
+          title: trans(`beatmaps.discussions.message_type_title.${typeText}`),
```

Comment:
title tooltip behaviour is rather bugged in chrome when the button is disabled. Wrap it with a div or something and put the title there (it's still kind of buggy but at least it's less bad).